tests/cephadm/test_add_host_to_maintenance_using_flags.py

```diff
@@ -0,0 +1,63 @@
+from ceph.utils import get_node_by_id
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+from cli.ops.host import host_maintenance_enter, host_maintenance_exit
+
+
+def run(ceph_cluster, **kw):
+    """
+    Create a pool for rbd application that has
+    size and min_size 2 and pool_type replicated.
+    Force a host with OSDs to enter maintenance mode
+    using flags --force and --yes-i-really-mean-it
+    Validate the logs for no errors
+    Remove the host from maintenance
+    """
+    config = kw["config"]
+    pool_config = config.get("pool_config")
+    pool_name = pool_config["pool_name"]
+    size = pool_config["size"]
+    min_size = pool_config["min_size"]
+    application = pool_config["app_name"]
+    instance = config.get("node")
+    instance_name = get_node_by_id(ceph_cluster, instance).hostname
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Create a pool
+    out = CephAdm(installer).ceph.osd.pool.create(**pool_config)
```

Comment:
With this code, command executed ?

Reply:
Hey @vamahaja I am unable to execute the test as the RHOSD is full, will update the PR with execution logs once its available.